### PR TITLE
chore: new cli does now have a nitric.yaml file

### PR DESCRIPTION
**Pull request by raksiv**
Remove statement about yaml files.

### CODE REVIEW

File: docs/support/upgrade.md [hunk 1]
```diff
@@ -225,5 +225,3 @@ publicApi.post('/examples', async (ctx) => {
   await examples.doc(name).set({ name, description });
 });
 ```
-
-> The nitric.yaml and api.yaml are not required.
```
Comment (asalkeld) [inline docs/support/upgrade.md:229]: maybe the api.yaml is not required?

Comment (jyecusch) [reply docs/support/upgrade.md:229]: @raksiv could you update with the new nitric.yaml details

Comment (raksiv) [reply docs/support/upgrade.md:229]: Added new yaml details, and a note that the api.yaml is not required.

**Review by asalkeld — APPROVED** [review]

File: docs/support/upgrade.md [hunk 2]
```diff
@@ -226,4 +226,12 @@ publicApi.post('/examples', async (ctx) => {
 });
 ```
 
-> The nitric.yaml and api.yaml are not required.
+> The api.yaml file is no longer required.
+
+> The nitric.yaml file now only contains basic meta information such as the project name and handler pathes.
```
Comment (davemooreuws) [inline docs/support/upgrade.md:231]: pathes to paths

**Review by davemooreuws — CHANGES_REQUESTED** [review]

File: docs/support/upgrade.md (hunk 2 above)
Comment (raksiv) [reply docs/support/upgrade.md]: updated.

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**